packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -139,6 +139,32 @@ depth = model.infer_image(raw_img) # HxW raw depth map in numpy
 	];
 };
 
+export const depth_pro = (model: ModelData): string[] => {
+	const installSnippet = `# Download checkpoint
+pip install huggingface-hub
+huggingface-cli download --local-dir checkpoints ${model.id}`;
+
+	const inferenceSnippet = `import depth_pro
```

Comment:
I wonder if we should make the snippet more minimal given there are instructions on the model card

Reply:
Since we have been trying to make snippets end-to-end - I think having a more complete snippet here would be better (even if it mimics the model card)

Reply:
Agree with @Vaibhavs10, lately I've seen PRs with more complete snippets and I like them (and there's enough space to show a standalone example).